examples/express/helpers.ts

```diff
@@ -0,0 +1,22 @@
+import fs from 'fs'
+import fetch from 'node-fetch'
+import extract from 'extract-zip'
+import path from 'path'
+
+const SOURCE_URL = 'https://d2m9b44i9bvctd.cloudfront.net/dist.zip'
+
+export const fetchStaticFiles = async (): Promise<void> => {
```

Comment:
(please correct me if i'm wrong) from what i understand, making the express app fetch the frontend files and serve them like this means this is a [traditional web server /SSR](https://app.gitbook.com/s/pBW92htBuXTrDYoKovQ2/~/changes/1/learn-the-basics/integration-patterns/web-server-and-ssr-front-end/~/comments/2PeVjMn4dKMUr84kiS2H) example

would creating a new example/ explaining how to modify this Express server to be a [BFF](https://app.gitbook.com/s/pBW92htBuXTrDYoKovQ2/~/changes/1/learn-the-basics/integration-patterns/back-end-for-spa-front-end-bff/~/comments/wQwRwhbQ4f0dBmuMYJvv) instead be something we want to consider doing?

Reply:
in the context of LifeHack, from what i know from my peers + what i've seen from hackathon submissions, i think students nowadays are a lot more familiar with a SPA Frontend + Backend than a MVC style web server 

if our immediate goal is to make sgID as easy to onboard as possible for LifeHack, creating an example for the BFF integration pattern might be quite important

Reply:
Resolved: Discussed offline - it's not a traditional web server even though the SPA is served from the same server (SPA is CSR, traditional web server is SSR)